Backend/src/app.controller.ts

```diff
@@ -8,132 +8,133 @@ import {
   @ApiOperation({
-    summary: 'Get element by id',
-    description: 'Gets the element at the requested index',
+    summary: "Get element by id",
+    description: "Gets the element at the requested index",
   })
   @ApiResponse({
     status: 200,
-    description: 'Element',
+    description: "Element",
   })
   @ApiResponse({
     status: 500,
-    description: 'Internal server error.',
+    description: "Internal server error.",
     type: HttpException,
   })
-  async getData(@Param('id') id: number) {
+  async getData(@Param("id") id: number) {
     try {
-      const result = this.appService.get(id)
-      return result
+      const result = this.appService.get(id);
+      return result;
     } catch (error) {
-      throw new HttpException(error.message, 500)
+      throw new HttpException(error.message, 500);
     }
   }
 
-  @Post('nfts')
+  @Post("nfts")
   @ApiOperation({
-    summary: 'Register NFT',
-    description: 'Registers a NFT',
+    summary: "Register NFT",
+    description: "Registers a NFT",
   })
   @ApiResponse({
     status: 200,
-    description: 'NFT registered',
+    description: "NFT registered",
   })
   @ApiResponse({
     status: 500,
-    description: 'Internal server error.',
+    description: "Internal server error.",
     type: HttpException,
   })
   createNFT(@Body() body: NftMetadataDto) {
-    const updatedObj = this.appService.createNft(body.metadata)
-    return updatedObj
+    const updatedObj = this.appService.createNft(body.metadata);
+    return updatedObj;
   }
 
-  @Post('nfts/:id/image')
+  @Post("nfts/:id/image")
   @ApiOperation({
-    summary: 'Register NFT file',
-    description: 'Registers a file in the database',
+    summary: "Register NFT file",
+    description: "Registers a file in the database",
   })
   @ApiResponse({
     status: 200,
-    description: 'File registered',
+    description: "File registered",
   })
   @ApiResponse({
     status: 500,
-    description: 'Internal server error.',
+    description: "Internal server error.",
     type: HttpException,
   })
-  @ApiConsumes('multipart/form-data')
+  @ApiConsumes("multipart/form-data")
   @ApiBody({
     schema: {
-      type: 'object',
+      type: "object",
       properties: {
         file: {
-          type: 'string',
-          format: 'binary',
+          type: "string",
+          format: "binary",
         },
       },
     },
   })
-  @UseInterceptors(FileInterceptor('file'))
+  @UseInterceptors(FileInterceptor("file"))
   uploadNFTFile(
     @UploadedFile() file: Express.Multer.File,
-    @Param('id') id: number,
+    @Param("id") id: number
   ) {
     const fileData = new FileDataDto(
       file.originalname,
       file.mimetype,
       file.filename,
-      file.size,
-    )
-    this.appService.pushFileToNft(id, fileData)
-    const updatedObj = this.appService.saveToIpfs(id)
-    return updatedObj
+      file.size
+    );
+    id = Number(id);
+    let p = this.appService.pushFileToNft(id, fileData);
```

Comment:
is there a reason to declare the p variable and do not use it?

Reply:
My bad.  I forgot to Remove.it was for debugging purposes